zkevm-circuits/src/super_circuit.rs

```diff
@@ -869,6 +869,52 @@ pub(crate) mod super_circuit_tests {
         block
     }
 
+    pub(crate) fn block_type2_1tx() -> GethData {
```

Comment:
For me, it's better to name `dynamic fee tx` rather than `type2 tx`.

```suggestion
    pub(crate) fn block_1dynamic_fee_tx() -> GethData {
```